docs/index.rst

```diff
@@ -66,10 +66,10 @@ KSQL Server
     The KSQL server runs the engine that executes KSQL queries. This includes processing, reading, and writing data to and from
     the target Kafka cluster.
 
-    Servers can run in containers, virtual machines, and bare-metal machines. You can add or remove multiple servers in the
-    same resource pool to elastically scale query processing in or out. You can use different resource pools to achieve workload
-    isolation.
+    KSQL servers form KSQL clusters and can run in containers, virtual machines, and bare-metal machines. You can add or
+    remove multiple servers to/from the same KSQL cluster during live operations elastically scale query processing in
```

Comment:
you can replace `to/from` with `in`: `You can add or remove servers in the same KSQL cluster...`